tools/format.py

```diff
@@ -19,4 +19,4 @@
 run(["yapf", "-i"] + glob("tools/*.py"))
 run(["node", prettier, "--write"] + glob("js/*.js") + glob("js/*.ts") +
     ["tsconfig.json"] + ["tslint.json"])
-run(["rustfmt", "-f", "--write-mode", "overwrite"] + glob("src/*.rs"))
+run(["rustfmt", "--write-mode", "overwrite"] + glob("src/*.rs"))
```

Comment:
This breaks it for me again, I can't figure out how to get rustfmt to work without switching to the nightly toolchain.
They appear to try very hard to make it impossible.
Maybe pick up RUSTFMT_FLAGS from the environment so I can use that?

Reply:
It looks like you should already be able to use RUSTFMT_FLAGS
https://github.com/ry/deno/blob/d160de7f445e5d79883f2e8d97584b8afd009cc7/tools/util.py#L7-L13

Reply:
Ah, sorry, I thought you meant that RUSTFMT_FLAGS was an already recognized option... I added another commit that does as you asked